tox.ini

```diff
@@ -0,0 +1,18 @@
+[tox]
+envlist = py34, lint
+
+[testenv]
+setenv =
+    PYTHONPATH = {toxinidir}:{toxinidir}/homeassistant
+commands = py.test {posargs}
+     flake8 --exclude www_static homeassistant
+; If you want to make tox run the tests with the same versions, create a
+; requirements.txt with the pinned versions and uncomment the following lines:
```

Comment:
I don't understand what this comment means. The lines below are already uncommented?